domains/juniper.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "juniper_is_lost_",
```

username doesnt match github username
```suggestion
    "username": "juniperislost",
```